cutil/kubeadm/token.go

```diff
@@ -16,25 +16,20 @@ package kubeadm
 
 import (
 	"fmt"
-	"math/rand"
-	"time"
+	"kubicorn/cutil/rand"
```

Comment:
You have a mistake here, you have to change it to `github.com/kris-nova/kubicorn/cutil/rand`.